code/game/objects/items/stacks/stack.dm

```diff
@@ -236,8 +236,13 @@
 	return
 
 /obj/item/stack/attackby(obj/item/W, mob/user, params)
-	if(istype(W, src.type))
+	if(istype(W, src.type) || (istype(W, /obj/item/stack/cable_coil) && istype(src, /obj/item/stack/cable_coil)))
 		var/obj/item/stack/S = W
+
+		if(S.amount >= S.max_amount && !isrobot(user))
```

Comment:
Added a nice little message to let the non-robot user know if their stack was already full